client/src/components/Overview/ProductDetails 2.jsx

```diff
@@ -0,0 +1,37 @@
+import React, {useState, useEffect} from 'react';
```

Comment:
It looks like the same situation with Product Details and Style Selector potentially being duplicates.

Reply:
Dont know how that happened. Just deleted it.